app/scripts/controllers/deployment.js

```diff
@@ -42,6 +43,14 @@ angular.module('openshiftConsole')
         title: $routeParams.deployment
       }
     ];
+    var buildsVersion = APIService.getPreferredVersion('builds');
+    var replicaSetsVersion = APIService.getPreferredVersion('replicasets');
+    var limitRangesVersion = APIService.getPreferredVersion('limitranges');
+    var imageStreamsVersion = APIService.getPreferredVersion('imagestreams');
+    $scope.deploymentsVersion = APIService.getPreferredVersion('deployments');
+    $scope.eventsVersion = APIService.getPreferredVersion('events');
+    $scope.horizontalPodAutoscalersVersion = APIService.getPreferredVersion('horizontalpodautoscalers');
+
     $scope.healthCheckURL = Navigate.healthCheckURL($routeParams.project,
                                                     "Deployment",
                                                     $routeParams.deployment,
```

Comment:
Need to add group to the health check URL.

Reply:
I think I missed something:

```JavaScript 
// group is 4th param:
healthCheckURL: function(projectName, kind, name, group) {
```
Looks like we are sending it `apps`?  Generated url: `https://localhost:9000/dev-console/project/foo/edit/health-checks?kind=Deployment&name=deployment-example&group=apps`

Reply:
Yeah, I think you can add `$scope.deploymentsVersion.group` as the last param, then it will always be the same as everything else.

I see we were passing `apps` before, which is right, but it's better to use the value from preferred version.

Reply:
ah, gotcha. will update.